### PR TITLE
[minor] mcomix and atril update in rifle

### PR DESCRIPTION
**Pull request by pirate486743186**
adding rifle.conf

atril can open cbr,cbz too

mcomix can open rar, zip too, set to low priority, it's not an archive manager, it's a comic book viewer.

### CODE REVIEW

File: ranger/config/rifle.conf [hunk 1]
```diff
@@ -178,6 +178,7 @@ ext mobi, has ebook-viewer, X, flag f = ebook-viewer -- "$@"
 ext cb[rz], has qcomicbook, X, flag f = qcomicbook "$@"
 ext cb[rz], has mcomix,     X, flag f = mcomix -- "$@"
 ext cb[rz], has zathura,    X, flag f = zathura -- "$@"
+ext cb[rz], has atril,    X, flag f = atril -- "$@"
```
Comment (toonn) [inline ranger/config/rifle.conf:181]: Is this MATE's atril? That looks like a much more general document viewer so maybe it should be added for pdf and djvu and other formats it supports too?

File: ranger/config/rifle.conf [hunk 2]
```diff
@@ -227,6 +228,7 @@ ext ace, has unace = unace l "$1" | less
 ext ace, has unace = for file in "$@"; do unace e "$file"; done
 ext rar, has unrar = unrar l "$1" | less
 ext rar, has unrar = for file in "$@"; do unrar x "$file"; done
+ext rar|zip, has mcomix,     X, flag f = mcomix -- "$@"
```
Comment (toonn) [inline ranger/config/rifle.conf:231]: Let's have this in the Documents section of rifle.conf, I think that makes more sense.

**Review by toonn — CHANGES_REQUESTED** [review]
Couple suggestions, do you agree?

File: ranger/config/rifle.conf (hunk 2 above)
Comment (pirate486743186) [reply ranger/config/rifle.conf:231]: you get a lot of comic archives as .rar or .zip
This is not pointless

File: ranger/config/rifle.conf (hunk 1 above)
Comment (pirate486743186) [reply ranger/config/rifle.conf:181]: i only put in what i tested

Comment (pirate486743186) [reply ranger/config/rifle.conf:181]: atril with pdf and djvu is already in.....

Comment (toonn) [reply ranger/config/rifle.conf:181]: You're right, I didn't check closely enough.

File: ranger/config/rifle.conf (hunk 2 above)
Comment (toonn) [reply ranger/config/rifle.conf:231]: That's not what I said. Please reread my remark carefully, I'm only asking to move the condition into the cb[rz] rule, nothing more. Having it there would make other people test whether their favorite comic readers support `.zip` and `.rar`.

Comment (pirate486743186) [reply ranger/config/rifle.conf:231]: it's meant to be low priority, it's not a general archive manager
what you say, will make it come first with rar/zip files.

Comment (pirate486743186) [reply ranger/config/rifle.conf:231]: well, it doesn't work with atril, but works with qcomicbook
didn't test zathura

**Comment by pirate486743186** [comment]
i also added qcomicbook and zathura in the archives, like mcomix.

**Comment by pirate486743186** [comment]
maybe some one should merge this, before you get conflicts?

**Comment by toonn** [comment]
Conflicts aren't a big deal. Thanks for the ping though I didn't get any notification of your changes or I must've missed it. I'll take a look as soon as I find some time.

**Review by toonn — APPROVED** [review]
LGTM

**Pull request merged**